### DISCUSSION TITLE
可以通过 unbound_custom.conf 来自定义 Redis 服务地址吗？

### DISCUSSION THREAD

**Discussion by jakeslee** [kkkgo/PaoPaoDNS#176] [问答和帮助] [answered]
我想统一部署一套 Redis 集群，paopaodns 部署多个实例连一套集群使用。可以通过 unbound_custom.conf 覆盖 unbound.conf 里访问 Redis 的配置来实现连外部 Redis 吗？

**Comment by kkkgo** [comment, marked as answer]
可以的，不过需要注意使用了两个redis-logical-db分别是0和1

**Reply by kkkgo** [reply]
递归缓存只使用redis-logical-db 0，而redis-logical-db 1实际仅用于dnscrypt缓存，不会被unbound_custom.conf定义。
